### ISSUE TITLE
Using Environment Variables in codegen

### ISSUE THREAD

**Issue by mandyMooreFan** [kxbmap/sbt-jooq#11]
I have a question about using environment variables in the jooq-config file.  Or passing those values into the jooq-config file from SBT.  I'm not sure if I missed how to do this or if I missed something really easy in your example / test files. 

If i remove "${env.JDBC_URL}" from below and put the string values it works just fine.

jooq-codegen.xml 
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<configuration xmlns="http://www.jooq.org/xsd/jooq-codegen-3.10.0.xsd">

    <!-- JDBC connection parameters -->
    <jdbc>
        <driver>com.mysql.jdbc.Driver</driver>
        <url>${env.JDBC_URL}</url>
        <user>${env.JDBC_USERNAME}</user>
        <password>${env.JDBC_PASSWORD}</password>
    </jdbc>

    <!-- Generator parameters -->
    <generator>
        <database>
            <name>org.jooq.util.mysql.MySQLDatabase</name>
            <includes>.*</includes>
            <schemata>
                <schema>
                    <inputSchema>public</inputSchema>
                    <outputSchema>public</outputSchema>
                </schema>
            </schemata>
        </database>
        <target>
            <packageName>com.public.database</packageName>
            <directory>target/scala-2.12/src_managed/main/java</directory>
        </target>
    </generator>
</configuration>
```

build.sbt
```
val jdbcUrl = System.getenv("JDBC_URL")
val jdbcUsername = System.getenv("JDBC_USERNAME")
val jdbcPassword = System.getenv("JDBC_PASSWORD")

lazy val database = project.in(file("database"))
  .enablePlugins(JooqCodegen)
  .settings(
    libraryDependencies ++= Seq("runtime", "jooq").map { conf =>
      "mysql" % "mysql-connector-java" % "5.1.16" % conf
    },
    jooqVersion := "3.10.1",
    autoJooqLibrary := true,
    jooqGroupId := "org.jooq",
    jooqCodegen := jooqCodegen.dependsOn(flywayMigrate in schema).value,
    jooqCodegenConfigFile := Some(file("jooq-codegen.xml")),
    jooqCodegenStrategy := CodegenStrategy.Always
  )

lazy val schema = project.in(file("database/schema"))
  .settings(
    flywayUrl := jdbcUrl,
    flywayUser := jdbcUsername,
    flywayPassword := jdbcPassword,
    flywaySchemas := Seq("public"),
    flywayLocations := Seq("classpath:db/migration"),
    libraryDependencies += "mysql" % "mysql-connector-java" % "5.1.16" % "runtime"
  )

```

**Comment by kxbmap** [comment]
Thank you for using the plugin.
This feature is already implemented in master, but not yet released.
I will release next version in few days.

**Comment by mandyMooreFan** [comment]
Thanks so much for the fast response.

**Comment by kxbmap** [comment]
Sorry for too late. But now released plugin version 0.4.0 including this feature.
